src/layout/actions/layout.rs

```diff
@@ -617,6 +623,68 @@ impl LayoutTree {
             }
         }
     }
+
+    /// Adds gaps to all the views of the container at the `NodeIndex`
+    ///
+    /// If the `NodeIndex` doesn't point to a container, an error is returned.
+    fn add_gaps(&mut self, node_ix: NodeIndex) -> CommandResult {
+        let layout = match self.tree[node_ix] {
+            Container::Container { layout, .. } => layout,
+            _ => return Err(TreeError::UuidNotAssociatedWith(
+                ContainerType::Container))
+        };
+        let gap = registry::get_data("gap_size")
+            .map(RegistryGetData::resolve).and_then(|(_, data)| {
+                Ok(data.as_f64().map(|num| {
+                    if num <= 0.0 {
+                        0u32
+                    } else {
+                        num as u32
+                    }
+                }).unwrap_or(0u32))
```

Comment:
Yeah, I wasn't being clear there. I think if we need to document that behaviour any more it should probably live in a different place than the configuration file (some sort of proper docs on a site, for example). So I won't add that information, I'll do it later in a big doc sweep once we have more features to document (and a better place than the configuration file to document them)